api/controllers/article-controllers.js

```diff
@@ -9,3 +13,10 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getAllArticles = (req, res, next) => {
+  const { order = "DESC" } = req.params;
```

Comment:
This would come from req.query, changing the order would be like doing `/api/articles?order=DESC`. 

You don't need to do it for this ticket anyway, you can just default it (which you have done)